.github/workflows/spellcheck.yaml

```diff
@@ -0,0 +1,15 @@
+name: Spellcheck Action
+on: [push, pull_request]
+
+jobs:
+  build:
+    name: Spellcheck
+    runs-on: ubuntu-latest
+    steps:
+      # The checkout step
+      - uses: actions/checkout@v3
+      - uses: rojopolis/spellcheck-github-actions@0.27.0
+        name: Spellcheck
+        with:
+          source_files: "README.md"
```

Comment:
It would be nice if it could check all Markdown files, not just the README

Reply:
Hi... 
It is actually possible 😇
Should I do so?

Reply:
That would be really nice